README.md

```diff
@@ -169,6 +169,71 @@ If you want to use `xrpl.js` with React Native you will need to install shims fo
   import './shim'
   ...
   ```
+### Using xrpl.js with Vite React
+
+Similar to above, to get xrpl.js to work with Vite you need to set up a couple aliases in the vite.config.ts file. 
+
+1. If it's a fresh project you can use `npm create vite@latest` then choose the React and TypeScript options.
+
+2. Copy these settings into your `vite.config.ts` file.
+
+```
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import polyfillNode from 'rollup-plugin-polyfill-node'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  define: {
+    'process.env': {}
+  },
+  optimizeDeps: {
+    esbuildOptions: {
+
+        define: {
+          global: 'globalThis',
+        },
+        plugins: [
+            NodeGlobalsPolyfillPlugin({
+                process: true,
```

Comment:
We ended up not needing `process` here

Reply:
The code I have seems to run fine without `process` as well in the toy example - @ckniffen do you happen to remember what adding `process: true` does here?

Reply:
(If it isn't necessary I'll do a follow up PR to remove it)